Big-Sur.md

```diff
@@ -8,80 +8,32 @@
   ./fetch-macOS.py
   ```
 
-- Get `xar` software.
+- Unpack the InstallAssistant.pkg
 
   ```
-  cd ~  # modify as needed
-
-  git clone https://github.com/VantaInc/xar.git
-
-  cd xar/xar
-
-  ./autogen.sh
-
-  make
-  ```
-
-- Install dependencies for `darling-dmg`. (The following is for Debian/Ubuntu, different distros name packages differently.)
-
-  ```
-  apt install libxml2-dev libbz2-dev libfuse-dev cmake build-essential
-  ```
-
-- Get `darling-dmg` software.
-
-  ```
-  cd ~  # modify as needed
-
-  git clone https://github.com/darlinghq/darling-dmg.git
-
-  cd darling-dmg
-
-  # Install required deps - RTFM please ;)
-
-  cmake .
-
-  make
-  ```
-
-- Extract `SharedSupport.dmg` from the downloaded `InstallAssistant.pkg` file (around 9 GB).
-
-  ```
-  $ ~/xar/xar/src/xar -tf InstallAssistant.pkg  # list files
-  Bom
-  Payload
-  Scripts
-  PackageInfo
-  SharedSupport.dmg
-  ```
-
-  ```
-  $ ~/xar/xar/src/xar -xf InstallAssistant.pkg  # extract files
+  sudo apt install libarchive-tools #install unzipper for *.pkg
+  bsdtar xvf InstallAssistant.pkg # extract files
   ```
 
 - Extract `BaseSystem.dmg` from `SharedSupport.dmg`:
 
   ```
-  7z l SharedSupport.dmg  # This will list the files in the archive
+  7z x SharedSupport.dmg  # extract support dmg
 
   mkdir ~/stuff
-  ~/darling-dmg/darling-dmg SharedSupport.dmg ~/stuff  # Mounts SharedSupport.dmg to ~/stuff
-
-  $ 7z l ~/stuff/com_apple_MobileAsset_MacSoftwareUpdate/ee3ab6c04234b360dd8fca93c0ae49f957bf0843.zip  # The string of letters and numbers will vary
-  ...
-  2020-11-06 18:57:48 .....    652236311    646767350  AssetData/Restore/BaseSystem.dmg
+  sudo mount -oloop *.hfs ~/stuff
```

Comment:
@thenickdude 
Can't tell where this feature came from. I only know that I've not installed something in particular. Just used my Linux Mint 20 / Ubuntu Focal as it is to run the command.

here's the feedback from my mount if that helps:
`mount | grep hfs
$/mnt/sata/kvm/macOS-bigSur/pkg/shared/5.hfs on /mnt/sata/kvm/macOS-bigSur/pkg/shared/hfs type hfsplus (ro,relatime,umask=22,uid=0,gid=0,nls=utf8)`

To me it seems like Ubuntu can read HFS+ since over a decade > https://superuser.com/questions/84446/how-to-mount-a-hfs-partition-in-ubuntu-as-read-write

Reply:
@thenickdude 
I think my HFS support derives from this package which seems to be installed by default:
`apt show hfsprogs`

> Description: mkfs and fsck for HFS and HFS+ file systems
 The HFS+ file system used by Apple Computer for their Mac OS is
 supported by the Linux kernel.  Apple provides mkfs and fsck for
 HFS+ with the Unix core of their operating system, Darwin.
 .
 This package is a port of Apple's tools for HFS+ filesystems.
 .
 For users, HFS+ seems to be a good compromise to carry files between
 MacOS X and Linux Machines, as HFS+ doesn't suffer the problems of
 FAT32 like:
 .
  * huge space waste (in slack space as devices grow faster);
  * ability to create files that are more than 4GB in size (especially
    good for those working with multimedia and that need to carry large
    ISO files);
  * ability to use case preserving (and even sensitivity!);
  * ability to use uid's and gid's on the filesystem.
 .
 Users in general can enjoy such benefits since it is expected to have
 more HFS+ filesystems in use, as Apple has announced Macintoshes for
 ix86-64, besides the filesystem being already supported by PowerPC
 systems since the beginning.

Reply:
Ubuntu comes with almost all possible FS drivers by default.

```
$ find /lib/modules | grep hfs 
/lib/modules/5.7.0-050700-lowlatency/kernel/fs/hfsplus
/lib/modules/5.7.0-050700-lowlatency/kernel/fs/hfsplus/hfsplus.ko
/lib/modules/5.7.0-050700-lowlatency/kernel/fs/hfs
/lib/modules/5.7.0-050700-lowlatency/kernel/fs/hfs/hfs.ko
```

We can also recommend users to install the `hfsplus` package. I am not sure if it is required but installing it can't hurt.